cinder/volume/drivers/lightos.py

```diff
@@ -77,7 +77,12 @@
     cfg.IntOpt('lightos_api_service_timeout',
                default=30,
                help='The default amount of time (in seconds) to wait for'
-               ' an API endpoint response.')
+               ' an API endpoint response.'),
+    cfg.BoolOpt('lightos_use_ipacl',
+               default=True,
+               help='Use volume IP-ACL to restrict access by host,'
```

Comment:
I suggest:
"IPACL work in conjunction with the standard NVME ACL. A host must be in both the IPACL and the ACL of a volume to access that volume. Cinder always sets the volume ACL. If lightos_use_ipacl is set to True, Cinder will also add the host's IP addresses to a volume's IPACL. If set to False, any IP address may access the volume. The default is True."